src/silx/gui/plot/items/axis.py

```diff
@@ -156,15 +159,15 @@ def _getDataRange(self) -> tuple[float, float] | None:
         """Returns the range of data items over this axis as (vmin, vmax)"""
         raise NotImplementedError()
 
-    def isInverted(self):
+    def isInverted(self) -> bool:
         """Return True if the axis is inverted (top to bottom for the y-axis),
         False otherwise. It is always False for the X axis.
 
         :rtype: bool
         """
         return False
 
-    def setInverted(self, isInverted):
+    def setInverted(self, isInverted: bool):
         """Set the axis orientation.
 
         This is only available for the Y axis.
```

Comment:
Can you update/fix the docstring below:
`:param bool flag:` -> `:param isInverted:`